alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/ui/ProviderDistributionUIConfig.java

```diff
@@ -93,7 +93,8 @@ public ProviderDistributionUIConfig(final String label, final String urlName, fi
         final ConfigField projectNamePattern = TextInputConfigField.create(KEY_PROJECT_NAME_PATTERN, LABEL_PROJECT_NAME_PATTERN, DESCRIPTION_PROJECT_NAME_PATTERN, this::validateProjectNamePattern);
         final ConfigField configuredProject = EndpointTableSelectField.createSearchable(KEY_CONFIGURED_PROJECT, LABEL_PROJECTS, DESCRIPTION_PROJECTS, this::validateConfiguredProject)
                                                   .addColumn(new TableSelectColumn("name", "Project Name", true, true))
-                                                  .addColumn(new TableSelectColumn("description", "Project Description", false, false));
+                                                  .addColumn(new TableSelectColumn("description", "Project Description", false, false))
+                                                  .addDataFieldKey(ChannelDistributionUIConfig.KEY_PROVIDER_NAME);
```

Comment:
The naming of addDataFieldKey is a little confusing here.   It's used to create an initial empty model and then the EndpointTableSelectField will populate the additional fields from the current config so that you build a full model.  Should this be called a requiredFieldKey so the method would change to addRequiredFieldKey?